src/Engine/Rendering/BlurHUD.cpp

```diff
@@ -254,7 +253,8 @@ void BlurHUD::FillStencil(RenderScene& scene)
         if(!spriteJob->BlurBackground) {
             continue;
         }
-        glUniformMatrix4fv(glGetUniformLocation(shaderHandle, "M"), 1, GL_FALSE, glm::value_ptr(spriteJob->Matrix));
+        glm::mat4 MVP = VP * spriteJob->Matrix;
+        glUniformMatrix4fv(glGetUniformLocation(shaderHandle, "PVM"), 1, GL_FALSE, glm::value_ptr(MVP));
```

Comment:
In my opinion this is confusing naming. you have 3 different names:
MVP = VP*M, "PVM". possible to just use 1 variant?